ubiquibot-config-default.yml

```
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
+priority-labels:
+  - name: "Priority: 0 (Normal)"
+    weight: 1
+  - name: "Priority: 1 (Medium)"
+    weight: 2
+  - name: "Priority: 2 (High)"
+    weight: 3
+  - name: "Priority: 3 (Urgent)"
+    weight: 4
+  - name: "Priority: 4 (Emergency)"
+    weight: 5
+auto-pay-mode: true
+analytics-mode: true
```

I never understood this property. Shouldn't we always have analytics enabled?

```suggestion
analytics: true
```

---

@0xcodercrane